javascript-modal/styles.css

```diff
@@ -0,0 +1,47 @@
+.modal-backdrop {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: none;
+}
+
+.modal-dialog {
+  position: fixed;
```

Comment:
You shouldn't need to make both of these `position: fixed`. You should only need to give the parent `position: fixed`, and the content inside can just be styled normally using flexbox